tests/supply_chain/test_supply_chain.py

```diff
@@ -1364,6 +1368,444 @@ def test_consumer_receive_products_after_vlt_days(self) -> None:
         states = consumer_nodes[expected_frame:consumer_node_index:features].flatten().astype(np.int)
         self.assertEqual(required_quantity_2, states[IDX_RECEIVED])
 
+    def test_init_sku_dynamics_OneTimeSkuPriceDemandSampler(self):
+        """Test the reading of "store_001" SKU information of OneTimeSkuPriceDemandSampler."""
+        env = build_env("case_04", 100)
+        be = env.business_engine
+        assert isinstance(be, SupplyChainBusinessEngine)
+        Store_001: FacilityBase = be.world._get_facility_by_name("Store_001")
+        configs = Store_001.configs
+        world = be.world
+        sku = OneTimeSkuPriceDemandSampler(configs, world)
+        assert isinstance(sku, OneTimeSkuDynamicsSampler)
+        # Store_ 001 has two SKUs.
+        self.assertEqual(2, len(sku._cache[0]))
+        # The price of the first item of SKU with product ID 20 is 43.11954545454545 and the demand is 25.
```

Comment:
create a fake data file for this test (do not use the sample.csv directly), for an easier and simpler comparison.